### ISSUE TITLE
Improve badges by following shields.io conventions

### ISSUE THREAD

**Issue by jansegre** [sharelatex/github-latex-ci#19]
This is what the current badges look like:

 ![](https://www.sharelatex.com/github/repos/roboime/pfc-jan-bramigk/builds/latest/badge.svg)

While most other badges look like this:

[![Gratipay](http://img.shields.io/gratipay/shields.svg)](https://www.gratipay.com/Shields/) [![npm version](http://img.shields.io/npm/v/gh-badges.svg)](https://npmjs.org/package/gh-badges) [![build status](http://img.shields.io/travis/badges/shields.svg)](https://travis-ci.org/badges/shields) [![Dependency Status](https://david-dm.org/badges/shields.svg)](https://david-dm.org/badges/shields)

There is a [great project](https://github.com/badges/shields) ([shields.io](http://shields.io/)) which aims
to standardize badges and provide standard badges for other services.
I opened an [issue there](https://github.com/badges/shields/issues/391) to request them to serve badges
for github latex, but they said there is no API (for querying a build status I suppose).

I'd like to propose that sharelatex build badges follow [their proposed standard](https://github.com/badges/shields#solution):

![Badge design](https://raw.githubusercontent.com/badges/shields/master/spec/proportions.png)

I think it should be as simple as using [their templates](https://github.com/badges/shields/tree/master/templates) modulo licensing.

**Comment by jansegre** [comment]
This boils down to patching `app/views/badges/pdf.jade`, the new svg would look something this:

```jade
svg(xmlns="http://www.w3.org/2000/svg", width="#{it.widths[0]+it.widths[1]}", height="20")

	lineargradient#smooth(x2="0", y2="100%")
		stop(offset="0", stop-color="#bbb", stop-opacity=".1")
		stop(offset="1", stop-opacity=".1")

	mask#round
		rect(width="#{it.widths[0]+it.widths[1]}", height="20", rx="3", fill="#fff")

	g(mask="url(#round)")
		rect(width="#{it.widths[0]}", height="20", fill="#{it.escapeXml(it.colorA||'#555')}")
		rect(x="#{it.widths[0]}", width="#{it.widths[1]}", height="20", fill="#{it.escapeXml(it.colorB||'#4c1')}")
		rect(width="#{it.widths[0]+it.widths[1]}", height="20", fill="url(#smooth)")

	g(fill="#fff", text-anchor="middle", font-family="DejaVu Sans,Verdana,Geneva,sans-serif", font-size="11")
		text(x="#{(it.widths[0])/2}", y="15", fill="#010101", fill-opacity=".3") #{it.escapeXml(it.text[0])}
		text(x="#{(it.widths[0])/2}", y="14") #{it.escapeXml(it.text[0])}
		text(x="#{it.widths[0]+it.widths[1]/2-1}", y="15", fill="#010101", fill-opacity=".3") #{it.escapeXml(it.text[1])}
		text(x="#{it.widths[0]+it.widths[1]/2-1}", y="14") #{it.escapeXml(it.text[1])}

```

Which is an almost direct translation of [shields flat template](https://github.com/badges/shields/blob/master/templates/flat-template.svg?short_path=08dfc47).

Though it still needs a replacement for this `it` context object, it should be possible to get at least some acceptable hard coded values if I had a working environment of shields, and I would still need and environment for github-latex-ci to test this.

I may open a pull request with the untested changes for validation.

**Comment by jpallen** [comment]
Closed by #20
